src/Factory/FixtureInstantiator/FixtureInstantiatorForAll.php

```diff
@@ -4,15 +4,30 @@
 
 use Neusta\Pimcore\FixtureBundle\Fixture;
 
-class FixtureInstantiatorForAll implements FixtureInstantiator
+final class FixtureInstantiatorForAll implements FixtureInstantiator
```

Comment:
Maybe we should rename this like `FixtureInstantiatorForClassesWithoutConstructorParameters` or `InstantiatorForFixturesWithoutConstructorParameters` 🤔

Reply:
So long.. `FixtureInstatiatorConstructorLess` ?!